includes/MirahezeMagicHooks.php

```diff
@@ -29,14 +29,30 @@ public static function onCreateWikiCreation( $DBname ) {
 	}
 
 	public static function onCreateWikiDeletion( $dbw, $wiki ) {
-		if ( file_exists( "/mnt/mediawiki-static/{$wiki}" ) ) {
-			Shell::command( '/bin/rm', '-rf', "/mnt/mediawiki-static/{$wiki}" )->execute();
+		global $wgEchoSharedTrackingDB;
```

Comment:
Why are we introducing a global?

Reply:
Because that config is set in LS. I figured using this config would be better then hard coding mhglobal into here, I thought.

Reply:
Why would you be hardcoding mhglobal?

Reply:
Right now it uses $wgEchoSharedTrackingDB. Which is the global being used, unless you mean you want me to use ConfigRegistry?

Reply:
Yes, ConfigRegistry is the way to do it. global is being discouraged by MediaWiki and we removed it from all our extensions as well

Reply:
Ok, thanks. In MirahezeMagic however globals are used throughout which is why I did it here, so for this PR I will convert to ConfigRegistry and I'll do the rest of this extension to ConfigRegistry later then. Thanks!

Reply:
Done